modules/host/consts.go

```diff
@@ -45,13 +45,13 @@ var (
 	// keep the data even if the price of siacoin fluctuates, the price of raw
 	// storage fluctuates, or the host realizes that there is unexpected
 	// opportunity cost in being a host.
-	defaultCollateral = types.SiacoinPrecision.Mul64(500).Div(modules.BlockBytesPerMonthTerabyte) // 500 SC / TB / Month
+	defaultCollateral = types.SiacoinPrecision.Mul64(100).Div(modules.BlockBytesPerMonthTerabyte) // 100 SC / TB / Month
```

Comment:
lowered the default collateral because a lot of hosts complaining about how much collateral they were using.

this default will definitely be going back up. But while we're below 100-200 hosts I want to make becoming a host on the more friendly side - especially while there's actually not much uploading happening.